CHANGELOG.asciidoc

```diff
@@ -44,6 +44,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485]
 - Fix panic when a signal is delivered before the server is instantiated {pull}580[580]
 - Use type integer instead of number in JSON schemas where applicable {pull}641[641]
+- Set array item types to string in JSON schemas {pull}651[651]
```

Comment:
This should not be under 6.2 :)

Reply:
Same for `Use type integer instead of number in JSON schemas where applicable {pull}641[641]`

Reply:
woops 😬